meta-balena-common/recipes-connectivity/networkmanager/resin-files/98dhcp_ntp

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+#
+# Update NTP sources that are passed via DHCP.
+#
+
+INTERFACE=$1
+ACTION=$2
+
+SERVER_DIR=/run/chrony/
+SERVER_FILE=${SERVER_DIR}/added_dhcp_${INTERFACE}.sources
+
+if [ ! -d "$SERVER_DIR" ]; then
+	mkdir -p $SERVER_DIR
+	chmod 750 $SERVER_DIR
+fi
+
+case "$ACTION" in
+	up|dhcp4-change)
```

Comment:
Also, in the old implementation we did a `/usr/bin/chronyc online` to inform chrony we are online. This probably has to do with removing the `connectivity-change` event above but I don't fully understand how we are now covering this.

Reply:
This is now being covered in a separate '99onoffline_ntp' script.